tests/functional/api/ContentFileCest.php

```diff
@@ -0,0 +1,139 @@
+
+        $file = dispatch_now(CreateFile::image($image, 'Image', $language, $author, [
+            'info'        => 'info text',
+            'description' => 'My image',
+            'is_active'   => true,
+        ]));
+        $translation = dispatch_now(new AddFileTranslation($file, 'New translation', $language, $author,
+            ['description' => 'Description']
+        ));
+
+        dispatch_now(new SyncFiles($content, [$file->id => ['weight' => 3]]));
+
+        $I->sendGet(apiUrl("contents/$content->id/files"));
+
+        $I->seeResponseCodeIs(200);
+        $I->seeResponseIsJson();
+        $I->seeResponseJsonMatchesJsonPath('data[*]');
+        $I->seeResponseJsonMatchesJsonPath('links[*]');
+        $I->seeResponseJsonMatchesJsonPath('meta[*]');
+        $I->seeResponseContainsJson(
+            [
+                'data' => [
+                    [
+                        'id' => $file->id,
+                        'author_id' => $author->id,
+                        'name' => 'file',
+                        'extension' => 'jpg',
+                        'size' => 10240,
+                        'mime_type' => 'image/jpeg',
+                        'info' => 'info text',
+                        'is_active' => true,
+                        'created_at' => $file->created_at->toAtomString(),
+                        'updated_at' => $file->updated_at->toAtomString(),
+                        'translations'=> [
+                            [
+                                'author_id' => $author->id,
+                                'language_code' => 'en',
+                                'title' => 'New translation',
+                                'description' => 'Description',
+                                'created_at' => $translation->created_at->toAtomString(),
+                                'updated_at' => $translation->updated_at->toAtomString()
+                            ]
+                        ]
+                    ]
+                ]
+            ]
+        );
+    }
+
+    public function shouldSyncFilesWithContent(FunctionalTester $I)
+    {
+        $author   = factory(User::class)->create();
+        $language   = new Language(['code' => 'en']);
+        $content = dispatch_now(CreateContent::content('Content title', $language, $author, [
+            'body'      => 'Content body',
+            'is_active' => true
+        ]));
+
+        $image  = UploadedFile::fake()->image('file.jpg')->size(10);
+
+        $file = dispatch_now(CreateFile::image($image, 'Image', $language, $author, [
+            'info'        => 'info text',
+            'description' => 'My image',
+            'is_active'   => true,
+        ]));
+        $translation = dispatch_now(new AddFileTranslation($file, 'New translation', $language, $author,
+            ['description' => 'Description']
+        ));
+
+        $I->sendPUT(apiUrl("contents/$content->id/files"), [
```

Comment:
Please add `weight` to check if it works with API since files will be sorted and send with weights

Reply:
added